src/FamilyFilter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Choredo;
```

Comment:
Consider moving this out of main namespace? Feels like an Entity Concern? Or Repository? Idk.